cypress/integration/mouse.spec.ts

```diff
@@ -357,3 +357,64 @@ describe("realMouseMove", () => {
       .should("eq", "1.8");
   });
 });
+
+describe("canvas drag with realMouseMove", () => {
+  beforeEach(() => {
+    cy.visit("./cypress/fixtures/canvas-drag-svg.html");
+  });
+  it("drags with realMouseMove in every position option", () => {
+    cy.get("body")
+      .realMouseDown()
+      .realMouseMove(20, 10)
+      .realMouseMove(30, 20)
+      .realMouseUp();
```

Comment:
I added this test for the default `topLeft` position in addition to the explicit `topLeft` position. We are expecting the marks to be next to each other as below.

![image](https://user-images.githubusercontent.com/24983797/154092568-26cf42c9-8a2e-4a87-ab46-8a5117577072.png)

Reply:
I actually don't understand how it is posisble

Reply:
Since `topLeft` is the default option, I thought it's good to add a test for the default `topLeft` and the explicit `topLeft`. Should I remove one of the tests or is there another issue?

Reply:
I refactored the tests I added and I think it's more clear now. Here is the logic
1. Draw `<polyline>`s with 2 points using `.realMouseMove` with all 9 explicit `positions` from Cypress 
2. Check that all `<polyline>`s are not overlapping (by `cy.click()`)
3. Draw a final `<polyline>` with `.realMouseMove` and no `position` specified. The default value of `topLeft` should cause this to overlap the first `<polyline>`.
  * Check that first and last `<polyline>`s have the same `points`
  * Check that all `<polyline>`s except for the first are not overlapping (by `cy.click()`)

Is this a reasonable testing strategy?